mmhuman3d/data/data_structures/smc_reader.py

```diff
@@ -420,27 +488,31 @@ def get_iphone_depth(self, iphone_id=0, frame_id=None, disable_tqdm=False):
                 Defaults to None.
             disable_tqdm (bool, optional):
                 Whether to disable the entire progressbar wrapper.
-                Defaults to False.
+                Defaults to True.
+            vertical (bool, optional):
+                iPhone assumes horizontal orientation
+                if True, convert data to vertical orientation
+                Defaults to True.
 
         Returns:
-            ndarray:
+            frames:
+                An ndarray in shape [frame_number, height, width, channels].
         """
-        if frame_id is None or type(frame_id) == list:
-            frames = []
-            frame_list = range(self.iphone_num_frames)
-            if frame_id:
-                frame_list = frame_id
-            for i in tqdm.tqdm(frame_list, disable=disable_tqdm):
-                frames.append(
-                    self.smc['iPhone'][str(iphone_id)]['Depth'][str(i)][()])
-            return np.stack(frames, axis=0)
-        else:
-            assert frame_id < self.iphone_num_frames,\
+        frames = []
+        if frame_id is None:
+            frame_list = range(self.get_iphone_num_frames())
+        elif isinstance(frame_id, list):
+            frame_list = frame_id
+        elif isinstance(frame_id, int):
```

Comment:
Do we have an else statement to capture invalid inputs?

Reply:
Fixed in https://github.com/open-mmlab/mmhuman3d/pull/76/commits/4d5dddf207b9419fd6ef5e266b1fb6f7dbf3cbb8.